applications/gaming.md

```diff
@@ -0,0 +1,20 @@
+---
+description: Machine translation for gaming
+---
+
+# Gaming
+
+Machine translation for **gaming** is the translation for in-game and on-screen text, game packs, **live chat**, customer support and how-to tutorials.
```

Comment:
live chat should be hyperlinked, not bolded.

However, I think this is a repeat of *Content types* below.

So here we can just leave it at

> Machine translation for **gaming** is for translating in-game and out-of-game content.

Reply:
I have updated all instances where there should be hyperlinks.
I didn't do it because I didn't know what pages to link.
I have copied your criterion now.